app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        android:screenOrientation="portrait"
```

Comment:
Wollen wir das so lassen? Wenn ja, können die screenOrientation Locks im Login und Register Fragment auch wieder weg.

Reply:
Finde dass die App nirgendwo für landscape geeignet ist bzw zu anfällig für crashes ist. Man kann das in Zukunft vllt nochmal überarbeiten und in die Issues schreiben